package.json

```diff
@@ -43,7 +43,8 @@
     "build-parity-electron": "cd packages/parity-electron && yarn build",
     "preelectron": "yarn build",
     "electron": "cd packages/fether-electron && yarn electron",
-    "lint": "semistandard 'packages/**/*.js' 'packages/**/*.ts' --parser babel-eslint",
+    "lint-files": "lintfiles() { [ $# -eq 0 ] && exit 0; echo 'Linting...' && prettier --write --ignore-path .gitignore --loglevel error $@ && semistandard $@ --fix --parser babel-eslint; }; lintfiles",
+    "lint": "yarn lint-files '**/*.{js,ts}'",
```

Comment:
I'd advocate `yarn lint` to not modify files (so no --write and --fix).

Reason: if somehow some ugly (but fixable) code goes to the CI, then the CI (which runs `yarn lint`) will fix this code locally and let the tests pass.

So I'd go for 2 scripts `yarn lint` and a `yarn lint:fix` or something like that.

Reply:
Good catch! I didn't know that the CI runs `yarn lint`.

Another possibility: continue using `yarn lint` to fix the files; don't let the CI run `yarn lint`, only `yarn test`, and in `yarn test` verify if the files are correctly linted. wdyt?

Also, side note: because we run Prettier and then override the code changes with Semistandard, we can only check if the code is compliant with Semistandard.

Reply:
Why not. What command would `yarn test` run to verify linting?

Btw: https://github.com/prettier/prettier/issues/150#issuecomment-284197974. I think it's possible to have a prettier-compliant check.

Reply:
`semistandard '**/*.{js,ts}' --parser babel-eslint`